bindata/cloud-network-config-controller/003-configmaps.yaml

```diff
@@ -11,8 +11,6 @@ apiVersion: v1
 metadata:
   namespace: openshift-cloud-network-config-controller
   name: trusted-ca
-  annotations:
-    networkoperator.openshift.io/create-only: "true"
   labels:
     config.openshift.io/inject-trusted-cabundle: "true"
 # will have CA-bundle injected by the CNO
```

Comment:
You remove the `create-only` annotation in this commit but don't update the CA-bundle injector until the next commit, so things are broken at this point.